skrub/_table_vectorizer.py

```diff
@@ -115,38 +115,31 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
 
     Applies a different transformation to each of several kinds of columns:
 
-    - numeric:
-        Floats, ints, and Booleans.
-    - datetime:
-        Datetimes and dates.
-    - low_cardinality:
-        String and categorical columns with a count of unique values smaller
-        than a given threshold (40 by default). Category encoding schemes such
-        as one-hot encoding, ordinal encoding etc. are typically appropriate
-        for columns with few unique values.
-    - high_cardinality:
-        String and categorical columns with many unique values, such as
-        free-form text. Such columns have so many distinct values that it is
-        not possible to assign a distinct representation to each: the dimension
-        would be too large and there would be too few examples of each
-        category. Representations designed for text, such as topic modelling
-        (GapEncoder) or locality-sensitive hashing (MinHash) are more
-        appropriate.
+    - numerical features: floats, integers, and booleans.
```

Comment:
also here the bullet point names `numeric` `datetime` `low_cardinality` `high_cardinality` corresponded exactly to the attribute names, which is more helpful than a paraphrase to do the mapping between the 2 IMHO